exonum-node/src/messages/types.rs

```diff
@@ -159,62 +148,57 @@ pub struct Propose {
     /// The validator id.
     pub validator: ValidatorId,
     /// The height to which the message is related.
```

Comment:
Shouldn't the description be changed?

```suggestion
    /// The epoch to which the message is related.
```

Reply:
Yeah, good catch 👍